.github/workflows/update-vendor.yaml

```diff
@@ -0,0 +1,55 @@
+name: Update vendored OSCAL content
```

Comment:
Per the project [style gude](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#yaml) please use 4 space indention.